webdriver/tests/classic/new_session/response.py

```diff
@@ -42,3 +42,10 @@ def test_capability_default_value(session, capability, default_value):
     assert isinstance(session.capabilities, dict)
     assert capability in session.capabilities
     assert session.capabilities[capability] == default_value
+
+
+def test_capability_user_agent(session):
+    assert isinstance(session.capabilities, dict)
+    assert "userAgent" in session.capabilities
+    assert type(session.capabilities["userAgent"]) == str
+    assert len(session.capabilities["userAgent"]) > 0
```

Comment:
All the browsers that have WebDriver implemented basically have a user agent value that is starting with `Mozilla/`. Probably we can have a check instead to verify that?

Reply:
Fair point. Lets leave it as is then.